crates/rpc-types/src/genesis.rs

```diff
@@ -262,7 +263,7 @@ mod tests {
                     holocene_time: Some(0),
                     isthmus_time: None,
```

Comment:
```suggestion
                    isthmus_time: Some(0),
```
isthmus should be enabled if jovian is enabled